docs/mkdocs.yml

```diff
@@ -24,7 +24,9 @@ plugins:
     - awesome-pages
 nav:
     - index.md
-    - mozdata:
+    - Cookbooks:
+        - Creating a derived dataset: cookbooks/creating_a_derived_dataset.md
+    - Datasets:
```

Comment:
We may still want to add other projects besides mozdata here. If we were to add marketing data, is there a way to nest that under here? I suspect it may be a better user experience to have each of these stay at the top-level, in which case I think it may be good to continue to have these headings be `mozfun` and `mozdata`, and we can rely on the index page to give the context of what those terms mean.

Reply:
It's possible to nest marketing data under `Datasets` when we get to it. 
I agree with @jklukas that it might be better to navigate the page if we have `mozdata` and `mozfun` at the top. However I feel like it's cleaner if we nest all projects under "Datasets". Using "Datasets" and "UDFs" instead of `mozdata` and `mozfun` would be more intuitive for users who are new to the repo. Maybe we could give more context about the project names such as `mozdata` and `mozfun` in the index under each nav item?

Reply:
Yep, I think subdividing between GCP project within each category makes sense -- and in fact we already do. 😄 This is how it looks currently:

![image](https://user-images.githubusercontent.com/20569/107555695-7b756880-6ba5-11eb-8224-f213b523e5b0.png)

Remember also that eventually at least some of (maybe all?) of this information will live in the glean dictionary.